astropy/wcs/tests/test_tab.py

```diff
@@ -59,3 +60,14 @@ def test_mixed_celest_and_1d_tab_roundtrip():
 
     pts = np.random.random((10, 3)) * [[2047, 2047, 127]]
     assert np.allclose(pts, w.wcs_world2pix(w.wcs_pix2world(pts, 0), 0))
+
+
+def test_wcstab_swapaxes():
+    # Crash on deepcopy of swapped -TAB axes reported in #13036.
+    # Fixed in #13063.
+    filename = get_pkg_data_filename('data/tab-time-last-axis.fits')
+    hdul = fits.open(filename)
+    w = wcs.WCS(hdul[0].header, hdul)
+    hdul.close()
```

Comment:
Why not use context manager?
```suggestion
    with fits.open(filename) as hdul:
        w = wcs.WCS(hdul[0].header, hdul)
```

Reply:
my understanding is that they do not do anything [for FITS files]. Nothing wrong with calling `open` and `close`.